extensions/wikia/Recirculation/RecirculationContent.php

```diff
@@ -27,23 +29,25 @@ class RecirculationContent implements JsonSerializable {
 	 * @param $publishDate
 	 * @param $author
 	 */
-	public function __construct($index, $url, $thumbnail, $title, $publishDate, $author) {
-		$this->index = $index;
-		$this->url = $url;
-		$this->thumbnail = $thumbnail;
-		$this->title = $title;
-		$this->publishDate = $publishDate;
-		$this->author = $author;
+	public function __construct( array $properties = [] ) {
+		foreach($properties as $key => $value){
+			$this->{$key} = $value;
```

Comment:
Let's save that in an old-school assoc. array instead of relying on magic like that.